orangecontrib/argument/miner/chunker.py

```diff
@@ -23,170 +19,273 @@
 from textblob import TextBlob
 
 
-class ArgumentChunker:
```

Comment:
Yes, and the only reason is that I don't see the benefits of following OOP in this case. Especially, in the previous design, I have the chunk data encapsulated inside the class, which also makes isolated testing difficult.

Reply:
Ok, makes sense. Honestly, I find it a bit inconsistent now because the `TopicModel` part is class-based. But I may be biased because I personally OOP even just for the purpose of grouping complex functionality, like all the loose functions that you have.

On the other hand, you are right that testing becomes a bit more complicated. That can serve as an occasion to test the entire class rather than the individual functions only, but it does require more effort indeed.

Reply:
I understand, and it also makes sense to do what you mentioned (organizing functions into a class). Then the easiest way to change is to have a class with all of those functions as either static or class functions, which is, from my view point, not necessary and even weirder then keeping them as module functions. For now, I would like to keep it this way, but still thanks for pointing it out!